core/src/apps/cardano/helpers/credential.py

```diff
@@ -183,6 +185,11 @@ def get_title(self) -> str:
             return ""
 
     def format(self) -> list[PropertyType]:
+        from ...common.paths import address_n_to_str
+        from . import bech32
+
+        pointer = self.pointer  # local_cache_attribute
```

Comment:
any reason this is `self_pointer` instead of `pointer`? did the simplify algorithm not catch this or did you make this change deliberately?

Reply:
I see `pointer = self.pointer`, there is not `self_pointer` in the current version.

I had to include `self_*` or `msg_*` in a small amount of cases where "*" was colliding with already existing symbol name.